conseil-common/src/main/scala/tech/cryptonomic/conseil/common/config/Platforms.scala

```diff
@@ -137,20 +137,44 @@ object Platforms {
     override val platform: BlockchainPlatform = Bitcoin
   }
 
+  /** configurations to describe a ethereum node */
+  final case class EthereumNodeConfiguration(
```

Comment:
Should we define a common data for this config?

Reply:
do you mean some trait?

Reply:
No, just that it's not chain dependent... it just describes any internet resource path on a host with port

Why having different classes if they all essentially describe an rpc endpoint?

Reply:
good point, I've changed it to the `java.net.URL` I was influenced by the Tezos code :P

Reply:
Not exactly what I meant, but... 😄 
Hell, this is so simpler, why not?

I'm not sure how frequent you might want to only change a single element of the endpoint, so for now I'll keep it like that.

P.s. to be clear, I meant that the whole case class could be put in common between all the chains, since it didn't specify anything chain-specific

Reply:
ok :) ok, now I don't know if I did it right :) we'll see, I'll change it to case class if there are any problems in the future

Reply:
yes, I think we can always revert back to individual pieces if ever needed. I personally doubt that.